src/main/java/co/rsk/federate/config/PowHSMBookkeepingConfig.java

```diff
@@ -26,20 +26,20 @@ public BigInteger getDifficultyCap() {
 
     private static final Logger logger = LoggerFactory.getLogger(PowHSMBookkeepingConfig.class);
 
-    private static final String DIFFICULTY_TARGET_PATH = "bookkeeping.difficultyTarget";
-    private static final BigInteger DIFFICULTY_TARGET_DEFAULT = BigInteger.valueOf(3);
+    public static final String DIFFICULTY_TARGET_PATH = "bookkeeping.difficultyTarget";
+    public static final BigInteger DIFFICULTY_TARGET_DEFAULT = BigInteger.valueOf(3);
```

Comment:
This default is being delete in this other [PR](https://github.com/rsksmart/powpeg-node/pull/271)